File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ""
    tmp = list(map(str, s.split(" ")))
    for i in range(len(tmp)):
        if tmp[i] == '':
            answer += str(" ")
        else:
            answer += str(tmp[i][:1].upper() + tmp[i][1:].lower() + " ")
    answer = answer[:len(answer)-1]
    return answer

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        StringBuilder answer = new StringBuilder();
        String[] sp = s.toLowerCase().split("");
        boolean flag = true;

        for (String ss : sp) {
            if (flag) {
                answer.append(ss.toUpperCase());
            } else {
                answer.append(ss);
            }

            flag = ss.equals(" ");
        }

        return answer.toString();
    }
}
